docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -300,3 +300,29 @@ CodeChecker check -l ./compile_commands.json \
   -d gcc-double-free \ # disable gcc-double-free
   -o ./reports
 ```
+
+# Configuring the FB-Infer Analyzer
+
+As of CodeChecker 6.23, Codechecker can now execute the Facebook Infer Analyzer.
+The minimum version of GCC we support is 1.0.0.
```

Comment:
minimum version of facebook infer! is 1.1.0??

Reply:
fixed